src/preprocess.js

```diff
@@ -8,12 +8,12 @@ const schema = require("./schema");
 const isWhitespaceOnly = /^\s*$/;
 
 const filenames = Object.values(schema)
-  .map(table => table.filename)
-  .filter(filename => !!filename);
+  .map((table) => table.filename)
+  .filter((filename) => !!filename);
 
 function processLines(filename, callback) {
   return new Promise((resolve, reject) => {
-    const filePath = path.join(__dirname, "..", "data", filename);
+    const filePath = path.join(__dirname, "..", "processed", filename);
```

Comment:
The encoding converter now outputs into `processed`, because it corrupts the files if it tries to convert already-converted utf8 data to utf8. I couldn't get encoding detection to work reliably, so the next best thing is to separate the output. The rest of the processes here shouldn't corrupt the data if run twice.